mappings/net/minecraft/container/CartographyTableContainer.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS ayg net/minecraft/container/CartographyTableContainer
 	FIELD c inventory Lags;
 	FIELD d context Layk;
+	FIELD e itemTakingInProgress Z
```

Comment:
takingItem

Reply:
Hmm, I want to have the "this is currently happening" part expressed in there.
What do you think about `currentlyTakingfItem`? Too verbose and clear anyways, or okay?